doc/user_guide/statements/insert.md

```diff
@@ -0,0 +1,68 @@
+# INSERT 
+
+You can construct [`INSERT`](https://docs.exasol.com/sql/insert.htm) SQL statements using the `Insert` class.
+
+## Creating `INSERT` Statements
+
+Create an `INSERT` statement:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("tableName")
+                .values("value1","value2","value3");
+```
+
+Create an `INSERT` statement for specific fields:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("tableName")
+                .field("column1", "column2","column3")
+                .values("value1","value2","value3");
+```
+
+### Using Placeholders
+
+In SQL you can use value placeholders (`?`) in prepared statements. This allows you to add the values later in a safe way.
+
+you can add single placeholders like this:
+```java
+		        final Insert insert = StatementFactory.getInstance()
+                .insertInto("testTable")
+                .field("column1")
+                .valuePlaceholder();
+
+```
+or multiple in one statement:
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("testTable")
+                .field("column1", "column2","column3")
+                .valuePlaceholders(3);
+```
+
+### Using Value Tables
+
```

Comment:
Please add a short description of the examples you are giving.  In the same way as you did in the examples above